ec/resources/waf/managed_rule.go

```diff
@@ -364,11 +363,115 @@ func ResourceManagedRuleRead(ctx context.Context, d *schema.ResourceData, m inte
 
 func ResourceManagedRuleUpdate(ctx context.Context, d *schema.ResourceData, m interface{}) diag.Diagnostics {
 	var diags diag.Diagnostics
-	return diags
+
+	accountNumber := d.Get("account_number").(string)
+	managedRuleID := d.Id()
+
+	log.Printf("[INFO] Updating WAF Managed Rule ID %s for Account >> %s", managedRuleID, accountNumber)
+
+	managedRuleRequest := sdkwaf.UpdateManagedRuleRequest{}
+
+	managedRuleRequest.Name = d.Get("name").(string)
+	managedRuleRequest.RulesetID = d.Get("ruleset_id").(string)
+	managedRuleRequest.RulesetVersion = d.Get("ruleset_version").(string)
+
+	if policies, ok := helper.ConvertInterfaceToStringArray(d.Get("policies")); ok {
+		managedRuleRequest.Policies = *policies
+	} else {
+		return diag.Errorf("Error reading 'policies''")
+	}
+
+	// Disabled Rules
+	disabledRules, err := ExpandDisabledRules(d.Get("disabled_rule"))
+
+	if err != nil {
+		return diag.Errorf("error parsing disabled_rule: %+v", err)
+	}
+
+	managedRuleRequest.DisabledRules = *disabledRules
+
+	// Rule Target Updates
+	ruleTargetUpdates, err := ExpandRuleTargetUpdates(d.Get("rule_target_update"))
```

Comment:
Perhaps we should be using `GetOk` like with `general_settings` below?

Reply:
@stevenpaz thank you for the feedback. I have included this in my followup refactor ticket for the work we discussed before. I verified with and without the presence of Rule Target Updates in the resource file that functionally the provider performs as expected and without error.